app/channels/chat_channel.rb

```diff
@@ -1,20 +1,14 @@
 class ChatChannel < ApplicationCable::Channel
-
   def subscribed
     stream_from "chat"
   end
 
-  def send_message(payload)
-    message = Message.new(user_id: user_id, text: payload["message"])
-    if message.save
-      ActionCable.server.broadcast "chat", message: render(message)
-    end
+  def receive(payload)
```

Comment:
przecież tu broadcastujesz, dlaczego to receive anie send?

Reply:
btw powinna być obsługa jak się nie zapisze

Reply:
w ogóle nie powinieneś tutaj tworzyć obiektu https://github.com/michalg-/ticketownik/blob/master/app/channels/tickets_channel.rb